cmd/init.go

```diff
@@ -49,7 +55,18 @@ var initCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		appRoot := metadata.AbsPath(path.Join(appDir, appName))
+
+		initDir, err := flags.GetString("dir")
+		if err != nil {
+			return err
+		}
+
+		path, err := genKsRoot(appName, appDir, initDir)
```

Comment:
What's your opinion on renaming `appDir` to `wd` for working directory? (since it's potentially no longer the app dir) and calling `initDir` the `appDir`?